pages/queries.py

```diff
@@ -66,7 +66,8 @@ def click_dropdown_menu(self, text=None) -> typing.Any:
         for item in (i for i in items if i.text in text):
             if text == "Fork":
                 item.click()
-                return QueryDetailPage(self.selenium, self.base_url)
+                page = QueryDetailPage(self.selenium, self.base_url)
+                return page.wait_for_page_to_load()
```

Comment:
Is there a difference in UI behavior between upstream and fork?

Reply:
No, I just forgot to call the ```wait_for_page_to_load()``` and noticed it was causing some intermittent failures

Reply:
Okay, great! 👍